subscription-service/src/main/resources/application.properties

```diff
@@ -31,3 +31,7 @@ application.authentication.enabled = false
 # application.graylog.host = localhost
 # application.graylog.port = 12201
 # application.graylog.source = stellio-int
+
+# Pagination config for query resources endpoints
+pagination.limit.default = 30
```

Comment:
it is cleaner to have our own keys under the `application` prefix, to better distinguish them from external properties brought by Spring and others